lib/tasks/data/seed.rake

```diff
@@ -17,55 +15,55 @@ namespace :data do
-                        ['LPS11', 'Lead Practitioners Range 11', 50476],
-                        ['LPS12', 'Lead Practitioners Range 12', 51639],
-                        ['LPS13', 'Lead Practitioners Range 13', 52930],
-                        ['LPS14', 'Lead Practitioners Range 14', 54250],
-                        ['LPS15', 'Lead Practitioners Range 15', 55600],
-                        ['LPS16', 'Lead Practitioners Range 16', 57077],
-                        ['LPS17', 'Lead Practitioners Range 17', 58389],
-                        ['LPS18', 'Lead Practitioners Range 18', 59857],
-                        ['LPS19', 'Lead Practitioners Range 19', 61341],
-                        ['LPS20', 'Lead Practitioners Range 20', 62863],
-                        ['LPS21', 'Lead Practitioners Range 21', 64417],
-                        ['LPS22', 'Lead Practitioners Range 22', 66017],
-                        ['LPS23', 'Lead Practitioners Range 23', 67652],
-                        ['LPS24', 'Lead Practitioners Range 24', 69330],
-                        ['LPS25', 'Lead Practitioners Range 25', 71053],
-                        ['LPS26', 'Lead Practitioners Range 26', 72810],
-                        ['LPS27', 'Lead Practitioners Range 27', 74615],
-                        ['LPS28', 'Lead Practitioners Range 28', 76466],
-                        ['LPS29', 'Lead Practitioners Range 29', 78359],
-                        ['LPS30', 'Lead Practitioners Range 30', 80310],
-                        ['LPS31', 'Lead Practitioners Range 31', 82293],
-                        ['LPS32', 'Lead Practitioners Range 32', 84339],
-                        ['LPS33', 'Lead Practitioners Range 33', 86435],
-                        ['LPS34', 'Lead Practitioners Range 34', 88571],
-                        ['LPS35', 'Lead Practitioners Range 35', 90773],
-                        ['LPS36', 'Lead Practitioners Range 36', 93020],
-                        ['LPS37', 'Lead Practitioners Range 37', 95333],
-                        ['LPS38', 'Lead Practitioners Range 38', 97692],
-                        ['LPS39', 'Lead Practitioners Range 39', 100072],
-                        ['LPS40', 'Lead Practitioners Range 40', 102570],
-                        ['LPS41', 'Lead Practitioners Range 41', 105132],
-                        ['LPS42', 'Lead Practitioners Range 42', 107766],
-                        ['LPS43', 'Lead Practitioners Range 43', 109366]].freeze
+PAYSCALE_DATA = [['MPS1', 'Main pay range 1'],
+                 ['MPS2', 'Main pay range 2'],
+                 ['MPS3', 'Main pay range 3'],
+                 ['MPS4', 'Main pay range 4'],
+                 ['MPS5', 'Main pay range 5'],
+                 ['MPS6', 'Main pay range 6'],
+                 ['UPS1', 'Upper pay range 1'],
+                 ['UPS2', 'Upper pay range 2'],
+                 ['UPS3', 'Upper pay range 3'],
+                 ['LPS1', 'Lead Practitioners range 1'],
```

Comment:
Should `Practitioners` also be down cased?

Reply:
No, this is the data as it was provided by Michael and it matches staging (and thus production)

<img width="684" alt="screen shot 2018-11-05 at 17 22 32 1" src="https://user-images.githubusercontent.com/159200/48014836-78b28000-e11f-11e8-90e3-fbf3dcc3a2f2.png">